src/game/Chat/Channel.cpp

```diff
@@ -333,7 +369,7 @@ void Channel::SetPassword(Player* player, const char* password)
 void Channel::SetModeFlags(Player* player, const char* targetName, ChannelMemberFlags flags, bool set)
 {
     // Restrict input flags to currently supported by this method
-    flags = ChannelMemberFlags(uint8(flags) & (MEMBER_FLAG_MODERATOR | MEMBER_FLAG_MUTED));
+    flags = ChannelMemberFlags(uint8(flags) & (MEMBER_FLAG_MODERATOR | MEMBER_FLAG_MIC_MUTED | MEMBER_FLAG_MIC_MUTED | MEMBER_FLAG_VOICED));
```

Comment:
```
MEMBER_FLAG_MIC_MUTED | MEMBER_FLAG_MIC_MUTED
```
Is this duplication intentional? Same thing on L1161.

Reply:
I guess not, probably first one should remain MEMBER_FLAG_MUTED